docs/001_develop/03_client-capabilities/006_charts/002_g2plot-chart/03_chart_html_api.mdx

```diff
@@ -0,0 +1,430 @@
+---
+title: 'Chart Declerative HTML API'
+sidebar_label: 'Chart Declerative HTML'
+id: chart-html-api
+keywords: [genesis, foundation, ui, chart, g2plot-chart, g2plot]
+tags:
+  - connected
+  - datasource
+  - chart
+  - g2plot-chart
+  - g2plot
```

Comment:
Can we expand these sections please with anything you can think of relating to the API. These terms aid searchabiliy, which is something we've been critisised for previously